atat_provisioning_wizard_api.yaml

```
@@ -384,6 +383,29 @@ paths:
           description: Successful operation
         '404':
           description: Task Order with the given number does not exist
+  '/taskOrderFiles/{taskOrderId}/download':
+    get:
+      tags:
+        - taskOrder
+      description: Gets a Task Order PDF
+      operationId: downloadTaskOrder
+      parameters:
+        - name: taskOrderId
+          in: path
+          description: Task Order ID
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/pdf:
+              schema:
+                type: string
+                format: binary
+        '404':
+          description: Task Order with the given ID does not exist or has not completed virus scanning
```

Perhaps something less specific?  Not sure I like my own wording here.

```suggestion
          description: Task Order with the given ID does not exist or is pending processing
```